tools/gen_struct_info.py

```diff
@@ -249,6 +249,12 @@ def inspect_headers(headers, cflags):
                                '-sBOOTSTRAPPING_STRUCT_INFO',
                                '-sSTRICT',
                                '-sASSERTIONS=0',
+                               # We cannot bootstrap in wasmfs mode, as we do so
+                               # using -nostdlib, but it requires malloc/free.
+                               # We don't need a filesystem when bootstrapping
+                               # anyhow, so there is no real reason to get this
+                               # working in that mode.
+                               '-sWASMFS=0',
```

Comment:
I don't think this is worth landed.  I think its fine for this code to have knowledge of the defaults.  

I think there are many settings that, if enabled by default, would cause this code here to break.   This code just needs to be kept up-to-date as the defaults change.

Reply:
Fair points. This is a setting we intend to flip the default on eventually, though, so I thought this could save us some effort at that future time. But I don't feel strongly, so closing.

Reply:
I imagine by the time we flip the default one the the requirements will likely be "Make sure gen_struct_info works with wasmfs".

Reply:
Perhaps, yeah.